src/node/history.h

```diff
@@ -235,6 +248,11 @@ namespace ccf
       nodes(nodes_)
     {}
 
+    void register_callback(std::string name, CallbackHandler func) override
+    {
+      callbacks[name] = func;
```

Comment:
Do we need a map of those? It seems to me that we're always going to have the same three?